### PR TITLE
fix(1084): Add Sonar env vars to guide

### PR DESCRIPTION
**Pull request by tkyi**
Related to https://github.com/screwdriver-cd/screwdriver/issues/1084

### CODE REVIEW

File: docs/user-guide/environment-variables.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ toc:
       active: true
     - title: Build Specific
       url: "#build-specific"
+    - title: Plugins
```
Comment (jithine) [inline docs/user-guide/environment-variables.md:12]: Can we call this `Services` ? Plugins gives Jenkins connotations.

Comment (tkyi) [reply docs/user-guide/environment-variables.md:12]: I think we call it plugins everywhere already
https://docs.screwdriver.cd/about/contributing/where-to-contribute, http://screwdriver.cd
Did we want to change everything?

Comment (jithine) [reply docs/user-guide/environment-variables.md:12]: Better not. I guess I can reason `Screwdriver` interfaces with various external `Services` via `Plugins`.

**Review by jithine — APPROVED** [review]
👍

File: docs/user-guide/environment-variables.md [hunk 2]
```diff
@@ -43,12 +45,21 @@ _Note: Environment variables set in one job cannot be accessed in another job. T
 | SD_TEMPLATE_VERSION | Version of the template the job is using (blank if not using template)|
 | SD_TOKEN | JWT token for the build |
 
+## Plugins
+These environment variables may or may not be available depending on what plugins are installed.
+
+#### Coverage (Sonar)
+| Name | Value |
+|------|-------|
+| SD_SONAR_AUTH_URL | Screwdriver API endpoint for authenticating against Sonar |
```
Comment (minzcmu) [inline docs/user-guide/environment-variables.md]: Shall we make it clear that this endpoint is to give you an access token against Sonar? By reading this line i will feel this is just like a login stuff

**Review by minzcmu — CHANGES_REQUESTED** [review]

**Review by minzcmu — APPROVED** [review]

**Pull request merged**